### ISSUE TITLE
False positive when multiplying against an untyped int constant

### ISSUE THREAD

**Issue by ksoichiro** [charithe/durationcheck#4]
Sample code:
```go
package dur

import "time"

const intervalMS = 100

var interval = intervalMS * time.Millisecond
```

Outputs:
```
dur.go:7:16: Multiplication of durations: `intervalMS * time.Millisecond`
```
